tbconnect/models.py

```diff
@@ -71,7 +71,12 @@ class TBCheck(models.Model):
     city = models.CharField(max_length=255)
     age = models.CharField(max_length=5, choices=AGE_CHOICES)
     gender = models.CharField(max_length=7, choices=GENDER_CHOICES)
-    location = models.CharField(max_length=255, validators=[geographic_coordinate])
+    location = models.CharField(
+        max_length=255, validators=[geographic_coordinate], null=True
+    )
+    city_location = models.CharField(
+        max_length=255, validators=[geographic_coordinate], null=True
+    )
```

Comment:
Why are these `null=True` if there `not None` validation in serializer?

Reply:
@cnsr only one of them is required. So on model level both can be none but I validate in the serializer that at least one is set.